docs/requirements.txt

```
@@ -1,4 +1,4 @@
-sphinx>=3.0, !=3.5  # fails with sphinx.ext.viewcode
+sphinx>=3.0, !=3.5, <4.0  # fails with sphinx.ext.viewcode  # fails with sphinx_paramlinks
```

```suggestion
sphinx>=3.0, <3.5  # fails with sphinx.ext.viewcode  # fails with sphinx_paramlinks
```
there is no other release in between :]